VEHICLE/N_0x8aa9180de2fedd45.md

```diff
@@ -1,15 +1,19 @@
 ---
 ns: VEHICLE
 ---
-## _0x8AA9180DE2FEDD45
+## EnableAircraftObstacleAvoidance
 
 ```c
 // 0x8AA9180DE2FEDD45 0xAE040377
-void _0x8AA9180DE2FEDD45(Vehicle vehicle, BOOL p1);
+void _0x8AA9180DE2FEDD45(Vehicle vehicle, BOOL enableobstacleavoidance);
```

Comment:
Rename `_0x8AA9180DE2FEDD45` to `EnableAircraftObstacleAvoidance`.
`enableobstacleavoidance` could also be simplified to `avoidObstacles`?